tests/main/reference/hvdc_HvdcPVDangling_HvdcPTanPhiDangling_diagrams/outputIIDM.xml

```diff
@@ -24,14 +24,14 @@
                 <iidm:bus id="_BUS___10_TN" v="399.97780496606896" angle="-0.0053007205545647987"/>
             </iidm:busBreakerTopology>
             <iidm:load id="_Load__10" name="LOAD  10" loadType="UNDEFINED" p0="500" q0="0" bus="_BUS___10_TN" connectableBus="_BUS___10_TN" p="500.02884066975162" q="0"/>
-            <iidm:vscConverterStation id="VSCStation1" name="VSCS1" voltageRegulatorOn="true" lossFactor="0" voltageSetpoint="400" reactivePowerSetpoint="0" bus="_BUS___10_TN" connectableBus="_BUS___10_TN" p="-100" q="20.031018988425391">
+            <iidm:vscConverterStation id="VSCStation1" name="VSCS1" voltageRegulatorOn="true" lossFactor="0" voltageSetpoint="399.97780496606896" reactivePowerSetpoint="0" bus="_BUS___10_TN" connectableBus="_BUS___10_TN" p="-100" q="20.031018988425377">
                 <iidm:reactiveCapabilityCurve>
                     <iidm:point p="-1011" minQ="-300" maxQ="300"/>
                     <iidm:point p="0" minQ="-600" maxQ="400"/>
                     <iidm:point p="1011" minQ="-300" maxQ="300"/>
                 </iidm:reactiveCapabilityCurve>
             </iidm:vscConverterStation>
-            <iidm:lccConverterStation id="LCCStation1" name="S1" lossFactor="0" powerFactor="0.90000000000000002" bus="_BUS___10_TN" connectableBus="_BUS___10_TN" p="-100" q="-0"/>
+            <iidm:lccConverterStation id="LCCStation1" name="S1" lossFactor="0" powerFactor="-1" bus="_BUS___10_TN" connectableBus="_BUS___10_TN" p="-100" q="1.2246467991473532e-14"/>
```

Comment:
same question as above

Reply:
this is already there in the master but not detected by the comparison script